Library/RewardCellProjectBackingStateType.swift

```diff
@@ -13,6 +14,11 @@ public enum RewardCellProjectBackingStateType: Equatable {
 
   static func state(with project: Project) -> RewardCellProjectBackingStateType {
     guard let backing = project.personalization.backing else {
+      if featurePostCampaignPledgeEnabled(), project.postCampaignPledgingEnabled,
+        project.isInPostCampaignPledgingPhase {
```

Comment:
nit: I don't think we need to check if the `project.postCampaignPledgingEnabled` - I don't think that can be false if it's in the post campaign pledging phase.